### ISSUE TITLE
Clean up code to facilitate ASL project

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/NGT-signbank#114]
- See Henri's work on the Finnish Signbank
- Note that we will at some point need the mobile templates for a public interface; keep those
- Look for 'Auslan' (Australian Sign lg.) or 'BSL' (British Sign Lg.) in file names and code; also look for 'ucl' (University College London, where the BSL Signbank is developed)
- Delete one folder with style sheets? (css = bootstrap_css?)
- Delete Projektor & superfish under /media? outdated media players / template
- /media/images: maps, one-handed, two-handed: all auslan/bsl-specific
- related: /templates/numbersigns, /templates/fingerspell
- folder /media/uclimages now contains Dutch image files, rename
- can /templates be merged with /signbank/dictionary/templates?

**Comment by Woseseltops** [comment]
Found in #32 : 

> There's a few field choice lists that are still in models.py that had better be moved to the Django admin field choices: RELATION_ROLE_CHOICES, DEFN_ROLE_CHOICES. (Found them in a search for some untranslated items, and noted that all the old choice lists are still embedded in models.py; better clean up now that they have been moved to the Django admin?

**Comment by ocrasborn** [comment]
Models.py contains unused field choices (moved to Django admin; can all be deleted?) and unused fields:

VARIOUS
* bsltf
* asltf
* aslgloss
* asloantf
* bslgloss
* bslloantf
* sedefinetf
* segloss

PHONOLOGY
* final_domhndsh
* final_subhndsh
* final_loc
* locsecond
* initial_secondary_loc
* final_secondary_loc
* initial_palm_orientation
* final_palm_orientation
* initial_relative_orientation
* final_relative_orientation
* absOriPalm
* absOriFing

**Comment by ocrasborn** [comment]
@Woseseltops , please complete #76 and #97 before moving on to the ASL labeled ones.

**Comment by ocrasborn** [comment]
See in the commits for https://github.com/Signbank/FinSL-signbank what Henri already did in this respect, much will apply to us as well.

**Comment by Woseseltops** [comment]
Note to self: search the whole project for 'legacy'

**Comment by Woseseltops** [comment]
Starting with the small/easy stuff:
* Deleted folders templates
* Deleted folders in the media folder: css, Projekktor, superfish, images/maps, images/one-handed, images/two-handed
* Deleted from bootstrap_templates: fingerspell, numbersigns

**Comment by Woseseltops** [comment]
More progress:
* Renamed bootstrap_css to css and bootstrap_templates to templates, and then fixed everything (?) that broke because of this.
* Searched for 'Auslan', 'BSL' and 'ucl', and removed it in most places, except where I thought the code could serve as an example for later.
* Images were stored in multiple locations. Moved all of that to media/images. uclimages is now deleted

For next time:
* Unsure what to do with auslan_media_prefix
* It looks like auslan.css is still being used; that should be fixed
* BSL_choices in models.py is still being used

**Comment by Woseseltops** [comment]
Fixed: 
* BSL_choices in models.py is still being used

Full list of things to do:
* Search for legacy
* Unsure what to do with auslan_media_prefix
* It looks like auslan.css is still being used; that should be fixed
* There are a lot unused hardcoded choices in models.py
* glossedit.js has a lot of old, unused code
* make all field choices dynamic
* look at Henri's work

**Comment by Woseseltops** [comment]
Done
* Search for legacy
* Unsure what to do with auslan_media_prefix
* It looks like auslan.css is still being used; that should be fixed
* There are a lot unused hardcoded choices in models.py
* glossedit.js has a lot of old, unused code

Full list of things to do:
* make all field choices dynamic
* look at Henri's work

**Comment by Woseseltops** [comment]
Full list of things to do:

* remove unused settings files
* separate settings files into Django general, Signbank general and language specific
* Delete all temporary ~ files from the repo
* completely make separate what is project specific, server specific and signlanguage specific
* make all field choices dynamic
* look at Henri's work

**Comment by Woseseltops** [comment]
Done:
* remove unused settings files
* separate settings files into Django general, Signbank general and language specific
* Remove unused settings

**Comment by Woseseltops** [comment]
I made an up-to-date todo list.

Quick:
* Delete all temporary ~ files from the repo
* Remove obsolete field 'init_text'
* Remove various symlinks from the media folder
* Rename scripts in bin folder
* Remove all unused imports

More effort:
* Rename media folder to static folder
* UserProfile should be in registration app
* Make the language choices for pages and menu items more dynamic (=remove references to english and dutch)

**Comment by vanlummelhuizen** [comment]
I did a quick search for 'auslan', 'bsl', 'ucl' and 'legacy'. 'auslan' and 'bsl' are still found it quite a number of places. Changing some things involves Django migrations which is not yet possible due to the Django and Python upgrades @Woseseltops is performing now. So, we need to revisit this issue later.

Note to self: the search command I used was

```grep -iR -P 'auslan|bsl|ucl|legacy' * | grep -v Binair | grep -v -P '^django' | grep -v -i -P 'jquery|bootstrap|migration' | less```
